src/com/sap/piper/Utils.groovy

```diff
@@ -2,36 +2,40 @@ package com.sap.piper
 
 import com.cloudbees.groovy.cps.NonCPS
 
-@NonCPS
-def getMandatoryParameter(Map map, paramName, defaultValue) {
 
-    def paramValue = map[paramName]
+class Utils implements Serializable {
```

Comment:
In this file it only adds this line, the other lines it only adds 4 white-spaces, but the diff provided by GitHub is not good in this case.

Reply:
Adding this line causes Utils#retrieveGitCoordinates() to fail since the echo statement cannot be found anymore.
The tests provided with this pull request works also fine without the changes in this class. What about submitting only the tests without making this an explicit class.

Reply:
I did fix it with script.echo